src/js/client/editprefset-viewport.js

```diff
@@ -0,0 +1,1001 @@
+        },
+        renderEditSidebar: {
+            func: "{that}.renderMarkup",
+            args: ["editWidgetSidebar", "{that}.options.templates.editPrefWidget", "{arguments}.0"]
+        },
+        editPspShow: {
+            funcName: "gpii.devpmt.editPspShow",
+            args: ["{that}", "{arguments}.0"]
+        },
+        editPspMemory: {
+            funcName: "gpii.devpmt.editPspMemory",
+            args: ["{that}", "{arguments}.0"]
+        },
+        editValueEvent: {
+            funcName: "gpii.devpmt.editValueEvent",
+            args: ["{that}", "{arguments}.0"]
+        },
+        editProductEnabled: {
+            funcName: "gpii.devpmt.editProductEnabled",
+            args: ["{that}", "{arguments}.0", "{arguments}.1", "{arguments}.2"]
+        },
+        downloadPrefset: {
+            funcName: "gpii.devpmt.downloadPrefset",
+            args: ["{that}"]
+        },
+        savePrefset: {
+            funcName: "gpii.devpmt.savePrefset",
+            args: ["{that}" /*, "{arguments}.0" */]
+        },
+        updateMetadataFromPrefs: {
+            funcName: "gpii.devpmt.updateMetadataFromPrefs",
+            args: ["{that}"]
+        },
+        initSettingTableWidgets: {
+            funcName: "gpii.devpmt.initSettingTableWidgets",
+            args: ["{that}"]
+        },
+        openAddProductDialog: {
+            funcName: "gpii.devpmt.openAddProductDialog",
+            args: ["{that}", "{arguments}.0"] // appId
+        },
+        toggleDevModeView: {
+            funcName: "gpii.devpmt.toggleDevModeView",
+            args: ["{that}", "{that}.model.devModeOn"]
+        },
+        lookupGenericPrefValue: {
+            funcName: "gpii.devpmt.lookupGenericPrefValue",
+            args: ["{that}", "{arguments}.0", "{arguments}.1"] // context, commonTerm
+        },
+        lookupProductPrefValue: {
+            funcName: "gpii.devpmt.lookupProductPrefValue",
+            args: ["{that}", "{arguments}.0", "{arguments}.1", "{arguments}.2"] // context, product, settingTerm
+        },
+        lookupPspMetadata: {
+            funcName: "gpii.devpmt.lookupPspMetadata",
+            args: ["{that}", "{arguments}.0", "{arguments}.1", "{arguments}.2", "{arguments}.3"]
+                    // type: "show" or "memory", context, term, product
+        },
+        onEditContext: {
+            funcName: "gpii.devpmt.onEditContext",
+            args: ["{that}", "{arguments}.0"] // event
+        },
+        onDeleteContext: {
+            funcName: "gpii.devpmt.onDeleteContext",
+            args: ["{that}", "{arguments}.0"] // event
+        },
+        updateFoundationSticky: {
+            funcName: "gpii.devpmt.updateFoundationSticky",
+            args: ["{that}"]
+        }
+    },
+    listeners: {
+        "onCreate.initialize": {
+            funcName: "gpii.devpmt.npsetInit",
+            args: ["{that}"] //"onCreate listener"]
```

Comment:
Remove the comment if it's no longer needed.

Reply:
removed.